src/library/scala/inline.scala

```diff
@@ -13,8 +13,16 @@
 package scala
 
 /**
- * An annotation on methods that requests that the compiler should try especially hard to inline the
- * annotated method. The annotation can be used at definition site or at callsite.
+ * An annotation for methods that the optimizer should inline.
+ *
+ * Note that by default, the Scala optimizer is disabled and no callsites are inlined. See
+ * `-opt:help` for information how to enable the optimizer and inliner.
+ *
+ * When inlining is enabled, the inliner will always try to inline methods or callsites annotated
```

Comment:
As an aside, I really wish we could deprecat/e/remove the callside inline, it makes correct incremental compilation impossible: https://github.com/sbt/zinc/issues/537

Reply:
When the inliner is enabled, the inliner heuristics select callsites for inlining where neither the callsite nor the callee method is annotated `@inline`. So enabling the inliner breaks incremental compilation anyway, the feature of annotating callsites doesn't make a difference.

I'll improve the documentation here to say that inlining only happens if the callee is included in the `-opt-inline-from` pattern.

Side-note: I'm working on an article about how to use the 2.12/2.13 inliner and how it works, that will be a good reference. It should be done very soon.